local_tf/codegen.py

```diff
@@ -0,0 +1,63 @@
+# depends on grpcio-tools
+
+from grpc.tools import protoc
+
+import sys
+import os
+import shutil
+import tempfile
+
+tensorflow_root         = sys.argv[1]
+tensorflow_serving_root = sys.argv[2]
+
+if not os.path.isdir(tensorflow_root) or not os.path.isdir(tensorflow_serving_root):
+    print "No tensorflow_root (%s) or no tensorflow_serving_root (%s)" % (tensorflow_root, tensorflow_serving_root)
+    sys.exit(1)
```

Comment:
Please add a bit of guidance on where those directories typically reside or what to do to get them.